Intersect.Client/Entities/Player.cs

```diff
@@ -1215,7 +1191,7 @@ public bool TryTarget()
                                         en.Value.X == x &&
                                         en.Value.Y == y &&
                                         !((Event) en.Value).DisablePreview &&
-                                        (!en.Value.IsStealthed() || Globals.Me.IsInMyParty(en.Value)))
+                                        !en.Value.IsStealthed() || en.Value is Player player && Globals.Me.IsInMyParty(player))
```

Comment:
Pretty sure that removing the parentheses here also changes the logic, so add the parentheses back around so it's like this:
`(!en.Value.IsStealthed() || en.Value is Player player && Globals.Me.IsInMyParty(player)))`

Reply:
Woops, missed that.

Reply:
Woops, missed that.